submissions/IhorOzerov/html-css-popup/styles.css

```diff
@@ -0,0 +1,115 @@
+.greatest:focus-within{
+    outline: 1px dotted red;
+}
+
+.visually-hidden {
+     position: absolute;
+     width: 1px;
+     height: 1px;
+     margin: -1px;
+     border: 0;
+     padding: 0;
+     clip: rect(0 0 0 0);
+     overflow: hidden;
+}
+
+.header-icons {
+     display: flex;
+     justify-content: flex-start;
+     align-items: center;
+     padding-left: 0;
+     margin: 0 0;
+}
+
+.h-image a{
+    text-decoration: none;
+    color: rgb(72, 94, 71);
+    
+}
+
+label.more-btn:hover, .h-image:hover{
+    cursor: pointer;
+}
+
+li {
+     list-style: none;
+}
+
+.h-image {
+     width: 30px;
+     margin: 15px;
+     
+}
+
+.visible-img {
+     width:55px;
+     height: 50px;
+}
+
+ul.menu{
```

Comment:
Avoid to use too specific CSS selectors [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)